plan/config.go

```diff
@@ -59,17 +61,31 @@ func ReadConfigFromEnviron() (*Config, error) {
 
 	var axes Axes
 	var behaviors Behaviors
+	filterMap := make(map[string][]Filter)
 	for _, e := range os.Environ() {
-		if strings.HasPrefix(e, axisKeyPrefix) {
+		switch {
+		case strings.HasPrefix(e, axisKeyPrefix):
 			axis := parseAxis(strings.TrimPrefix(e, axisKeyPrefix))
 			axes = append(axes, axis)
-		} else if strings.HasPrefix(e, behaviorKeyPrefix) {
+			break
```

Comment:
In Go switch statements, `break` is the default. You don't need to add these.

Your linter or `go vet` should have complained about this.